README.md

```diff
@@ -23,6 +24,19 @@ curl -X POST -H "Content-Type: application/json" --data "{}" http://localhost:88
 
 See the [spec](inst/schema/spec.md) for more details.
 
+## Updating data
+
+This will evolve as the upstream data changes and as our needs change. These scripts will create rds files that can be used to initialise the app. We'll store these either on mrcdata.dide.ic.ac.uk or as github release artefacts and pull them in fairly automatically during deployment.  They will expand into the actual mint database, which is much larger than the rds but faster to read.
+
+1. Acquire new data from the science team; this will come as a number of .rds files, the largest of which will be quite large.
+2. Copy these files onto a network-accessible share (e.g. for Rich `~/net/home/mint`)
+3. RDP to `fi--didex1` and copy these files into `C:\xampp\htdocs\mrcdata\mint\<date>` where `<date>` is YYYYMMDD (just to keep things tidy)
+4. Update the paths in `inst/data.json` to reflect the new data
+
+Note that the script will avoid downloading the files if they are already present in the `import` directory, so do not update files on the server without renaming them.
```

Comment:
Again unclear which script being referred to here. Maybe need to update explaining that this happens as part of docker build.